tests/workflows/error_handling_module_path/tests/test_workflow.py

```diff
@@ -0,0 +1,59 @@
+from vellum.workflows.errors.types import WorkflowErrorCode
+
+from tests.workflows.error_handling_module_path.workflow import CustomErrorWorkflow
+
+
+def test_non_vellum_node_error_returns_node_execution_error():
+    workflow = CustomErrorWorkflow()
```

Comment:
Add a second test that uses `VellumErrorWorkflow`, mocks the APINode.run method to raise an error, and assert that it's an internal error

Reply:
Actually, define the APINode without a `url` and mock the `APINode._validate` method to be a no-op